src/main/java/gregtech/api/capability/impl/MultiblockRecipeLogic.java

```diff
@@ -54,12 +55,40 @@ protected IMultipleTankHandler getOutputTank() {
     protected boolean setupAndConsumeRecipeInputs(Recipe recipe) {
         RecipeMapMultiblockController controller = (RecipeMapMultiblockController) metaTileEntity;
         if (controller.checkRecipe(recipe, false) &&
-            super.setupAndConsumeRecipeInputs(recipe)) {
+            multiBlockSetupAndConsumeRecipeInputs(recipe)) {
             controller.checkRecipe(recipe, true);
             return true;
         } else return false;
     }
 
+    //Logic Mostly copied from AbstractRecipeLogic, but with some additional checking for Multiblock output spacing to prevent voiding
+    protected boolean multiBlockSetupAndConsumeRecipeInputs(Recipe recipe) {
+        int[] resultOverclock = calculateOverclock(recipe.getEUt(), getMaxVoltage(), recipe.getDuration());
+        int totalEUt = resultOverclock[0] * resultOverclock[1];
+        IItemHandlerModifiable importInventory = getInputInventory();
+        IItemHandlerModifiable exportInventory = getOutputInventory();
+        IMultipleTankHandler importFluids = getInputTank();
+        IMultipleTankHandler exportFluids = getOutputTank();
+        return (totalEUt >= 0 ? getEnergyStored() >= (totalEUt > getEnergyCapacity() / 2 ? resultOverclock[0] : totalEUt) :
+            (getEnergyStored() - resultOverclock[0] <= getEnergyCapacity())) &&
+            MetaTileEntity.addItemsToItemHandler(exportInventory, true, recipe.getAllItemOutputs(exportInventory.getSlots())) &&
+            recipe.getAllItemOutputs(exportInventory.getSlots()).size() <= getFreeSlots(exportInventory) &&
```

Comment:
Also there is more elaborate logic required as you need to know number of slots you can put items in to and their capacity. 

For example when I build Implosion compressor. With LV Output bus (4 slots) and I let process 4x Amethyst Dust + 2x TNT => 3x Amethyst + 2x Tiny Pile of Dark Ashes it will only process 22x and then stop even if there is still capacity in output for another 10 runs.

![2020-12-05_19 45 37](https://user-images.githubusercontent.com/3093101/101260831-e6806a00-3732-11eb-822a-27107a59d939.png)
![2020-12-05_19 42 36](https://user-images.githubusercontent.com/3093101/101260829-e54f3d00-3732-11eb-8ccb-f6e3d55d8f7f.png)

Reply:
There has been a refactor to catch this case. Now the recipe output will attempt to merge into the existing stacks in the output bus before attempting to add the recipe output items to empty slots in the output bus. The above issue no longer exists, and the voiding issue is also solved.